ai/ai_model_building/army_builder.py

```diff
@@ -1,4 +1,5 @@
 from unit_parser import UNIT_LIST
+from utils import removekey
 
 
 def unit_binder(unit_name):
```

Comment:
A try except would be cleaner structure wise to deal with exception i think

Reply:
Good idea, updating it